test/spaces_reader/fetcher-spec.ts

```diff
@@ -42,7 +42,7 @@ describe('spaces_reader fetcher', () => {
             ['range query', {
                 query: {
                     q: 'date:[2017-09-23T18:07:14.332Z TO 2017-09-25T18:07:14.332Z}',
-                    size: 100
+                    size: 150
```

Comment:
@jsnoble I didn't 100% fully understand the `spaces_reader` `fetcher` tests here, so please consider whether I'm breaking anything.  Mostly I'm updating the `size` to account for the "expansion" (`count * 1.5`) that the fetcher now does.